package.json

```diff
@@ -28,6 +32,6 @@
   "dependencies": {
     "@ffmpeg/ffmpeg": "^0.11.6",
     "ffprobe-wasm": "^0.3.1",
-    "svelte-material-ui": "^7.0.0-beta.8"
+    "i": "^0.3.7"
```

Comment:
Yeah that surely must be a typo in some command sorry , removed that dependancy but also ran npm audit and it seems like cypress has some ongoing vulnerabilites, You can run the npm audit fix --force on them later because it might introduce some breaking changes, for now I won't run it.